crypto/evp_extra/p_hmac_asn1.c

```diff
@@ -121,6 +121,11 @@ const EVP_PKEY_ASN1_METHOD hmac_asn1_meth = {
     EVP_PKEY_HMAC,
     {0xff} /* placeholder oid */,
     0 /* oid_len */,
+
+    EVP_PKEY_HMAC,
+    "HMAC",
+    "AWS-LC HMAC method",
```

Comment:
OpenSSL seemed to remove the `hmac_asn1_meth` in favor of providers from 1.1.1 -> 3.0, so this is non-existent today.
* https://github.com/openssl/openssl/commit/2cf765e5a20762df1442c80cd2afc99e8bb1b823

1.1.1 defines this as "OpenSSL HMAC method" however, should we change this to align? 
* https://github.com/openssl/openssl/blob/OpenSSL_1_1_1-stable/crypto/hmac/hm_ameth.c#L95-L101